docs/user_guide/02_medrecord.md

```diff
@@ -153,81 +181,120 @@ When building a MedRecord, you may want to save it to create a persistent versio
 ```{literalinclude} scripts/02_medrecord_intro.py
 ---
 language: python
-lines: 58-59
+lines: 69-70
 ---
 ```
 
-## Printing Overview Tables
+:::{dropdown} Methods used in the snippet
+
+- [`to_ron()`](medmodels.medrecord.medrecord.MedRecord.to_ron){target="_blank"} : Writes the MedRecord instance to a RON file.
+- [`from_ron()`](medmodels.medrecord.medrecord.MedRecord.from_ron){target="_blank"} : Creates a MedRecord instance from a RON file.
+:::
 
-The MedModels MedRecord class is designed to efficiently handle large datasets while maintaining a standardized data structure that supports complex analysis methods. As a result, the structure within the MedRecord can become intricate and difficult to manage. To address this, MedModels offers tools to help keep track of the graph-based data. One such tool is the `print_node_overview` method:
+## Overview Tables
 
-```{literalinclude} scripts/02_medrecord_intro.py
+The MedRecord class is designed to efficiently handle large datasets while maintaining a standardized data structure that supports complex analysis methods. As a result, the structure within the MedRecord can become intricate and difficult to manage. To address this, MedModels offers tools to help keep track of the graph-based data. One such tool is the [`overview_nodes()`](medmodels.medrecord.medrecord.MedRecord.overview_nodes){target="_blank"} method, which prints an overview over all nodes in the MedRecord.
+
+```{exec-literalinclude} scripts/02_medrecord_intro.py
 ---
 language: python
-lines: 38
+setup-lines: 1-45
+lines: 48
 ---
```

Comment:
i get this error here

![image](https://github.com/user-attachments/assets/f27d0f3d-2c23-4744-a5cd-f24412d6da0d)